src/helpers/shared.ts

```diff
@@ -19,3 +19,27 @@ export const shouldSkip = (): { skip: boolean; reason: string } => {
 };
 
 export const wait = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+export const calculateWeight = (label: LabelItem | undefined): number => {
+  if (!label) return 0;
+  const matches = label.name.match(/\d+/);
+  const number = matches && matches.length > 0 ? parseInt(matches[0]) || 0 : 0;
+  if (label.name.toLowerCase().includes("priority")) return number;
+  if (label.name.toLowerCase().includes("hour")) return number * 0.125;
+  if (label.name.toLowerCase().includes("day")) return 1 + (number - 1) * 0.25;
+  if (label.name.toLowerCase().includes("week")) return number + 1;
+  if (label.name.toLowerCase().includes("month")) return 5 + (number - 1) * 8;
```

Comment:
I don't understand this calculation, how is weight related to time?

Reply:
I calculated it to adjust the value with the defaults.
1hour: 0.125
1day: 1
1 week: 2
2 weeks: 3
3 weeks: 4
1 month: 5
and made a rule for multiplier like 0.125, 0.25, 1, 8